extension/js/common/api/email-provider/gmail/gmail-parser.ts

```diff
@@ -165,7 +165,9 @@ export class GmailParser {
       const parts = payload.parts!; // eslint-disable-line @typescript-eslint/no-non-null-assertion
       // are we dealing with a PGP/MIME encrypted message?
       const pgpEncrypted = Boolean(
-        parts.length === 2 && contentType?.value?.startsWith('multipart/encrypted;') && contentType.value.includes('protocol="application/pgp-encrypted"')
+        parts.length === 2 &&
+          contentType?.value?.startsWith('multipart/encrypted') &&
+          (contentType.value.includes('protocol="application/pgp-encrypted"') || parts[0]?.mimeType?.includes('application/pgp-encrypted'))
```

Comment:
here let's check if mimeType is equal to `application/pgp-encrypted`, not just includes, so it'll look like this:
```ts
... || parts[0].mimeType === 'application/pgp-encrypted'
```

Reply:
Thanks, Roma - I'll update the checking with a more reliable one. By the way, looks like similar scenario happens on the iOS app where the `multipart/encrypted` with encrypted emails are not recognizable. While I can't promise I can fix the same issue there, I'm also interested in doing so. I'll be checking in android app too if they its capable of rendering such email and file GH issue regarding the issue if the issue exists there too.

Reply:
This issue doesn't exists on the FlowCrypt android app and they have similar detection made: https://github.com/FlowCrypt/flowcrypt-android/blob/e58d040f8eade675ca8c6bd18cb608d46bcdb44b/FlowCrypt/src/main/java/com/flowcrypt/email/extensions/jakarta/mail/PartExt.kt#L82

Reply:
@martgil I assigned iOS issue for you, but if you'll have any issues implementing the fix - please let me know

Reply:
Thanks for the opportunity to familiarize myself with the iOS app's codebase. I'll explore it and let you know if I have any questions.